twake/backend/node/src/services/channels/services/channel/activities/types.ts

```diff
@@ -0,0 +1,23 @@
+import { Channel } from "../../../../types";
+import User from "../../../../../services/user/entity/user";
+import { ChannelMember } from "../../../../../services/channels/entities";
+
+export type ActorType = { type: string; id: User["id"] };
```

Comment:
Did not know thet `User["id"]` was working here.

Reply:
You want me to change this to `string` or we keep this ?

Reply:
Keep it